RiotNSE/NSEMemoryStore.swift

```diff
@@ -86,6 +103,10 @@ class NSEMemoryStore: MXMemoryStore {
     //  Override and return a user to be stored on session.myUser
     override func user(withUserId userId: String) -> MXUser? {
         if userId == credentials.userId {
+            //  if myUser is set, return that
+            if let myUser = myUser {
+                return myUser
+            }
```

Comment:
The line below looks to be a duplicate of the last line of the method.

Reply:
Was not a duplicate actually, but it can be simplified as `MXMyUser` to `MXUser` conversion. Updated.

Reply:
ah yes.